plugins/plugin-docker/che-plugin-openshift-client/src/main/java/org/eclipse/che/plugin/openshift/client/kubernetes/KubernetesStringUtils.java

```diff
@@ -31,7 +31,9 @@ private KubernetesStringUtils() {}
    */
   public static String getNormalizedString(String input) {
     int end = Math.min(input.length(), MAX_CHARS - 1);
-    return input.substring(0, end);
+    String stringLessThan63chars = input.substring(0, end);
+    String normilizedString = removeSpecialCharactersInTheEnd(stringLessThan63chars);
```

Comment:
minor typo : normalized instead of normilized

Reply:
ah, thank you ;-)